src/clims/api/endpoints/process_assignments.py

```diff
@@ -16,11 +16,15 @@ def post(self, request, organization_slug):
         """
 
         # TODO-auth: Ensure that the user is only assigning samples that are under the organization
-        substances = request.data["substances"]
+
+        # Entities is a list of global ids (e.g. Substance-100)
+        entities = request.data["entities"]
```

Comment:
On a second thought, I think I like using "substances" more, rather than "entities". "Substances" communicate to a higher degree what a process is. Unpacking containers to get its contents seem to me like another concern that may be put somewhere else.

Reply:
Right. I have a feeling that if we allow users to assign whatever (not only substances) it makes sense to name it that way here. It would perhaps be confusing if we start supporting assigning containers to then call it substances.